jscomp/runtime/caml_splice_call.ml

```diff
@@ -38,6 +38,21 @@ let spliceApply : obj -> obj -> obj = [%raw{|function(fn,args){
   return fn.apply(null,applied)
 }|}] 
 
+let spliceNewApply : obj -> obj -> obj = [%raw{|function (ctor,args){
+  var i, argLen;
+  argLen = args.length
+  var applied = [null]
```

Comment:
There's a bunch of code duplication here and in the function above. I guess it's OK as there's no good way around that anyway.

Question: why is there a `null` here and not in the code above?

Reply:
Refer to: https://github.com/rescript-lang/rescript-compiler/pull/5364#discussion_r882332723

Reply:
> Refer to: [#5364 (comment)](https://github.com/rescript-lang/rescript-compiler/pull/5364#discussion_r882332723)

Would you add a comment above.